docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
       - ./traefik.toml:/traefik.toml
+      - ./certs:/certs
```

Comment:
Can you rebase? Your `docker-compose.yml` has containers that are not there anymore. Mentioning this, because ideally we should add this to both variations (default and full).